python/dgl/transform.py

```diff
@@ -1255,56 +1255,139 @@ def out_subgraph(g, nodes):
         ret.edges[etype].data[EID] = induced_edges[i].tousertensor()
     return ret
 
-def to_simple(g, return_counts='count', writeback_mapping=None):
-    """Convert a heterogeneous multigraph to a heterogeneous simple graph, coalescing
-    duplicate edges into one.
+def to_simple(g, return_counts='count', writeback_mapping=True, share_ndata=True):
+    r"""Convert a heterogeneous multigraph to a heterogeneous
+    simple graph, coalescing duplicate edges into one.
 
-    This function does not preserve node and edge features.
+    For a heterograph with multiple edge types, we can
+    treat edges corresponding
+    to each type as a separate graph and compute the
+    `simple` for each of them.
+
+    Writeback mappings can be returned for each edge
+    type subgraph. A writeback mapping is a tensor
+    recording the edge ids in the `simpled` graph for
+    each edges of the original graph. If the graph has
```

Comment:
How about "For edges left in a simple graph, a writeback mapping records their ids in the original graph."?